### ISSUE TITLE
SXSW Bars and Restaurants Point Layer

### ISSUE THREAD

**Issue by jaime-mckeown** [cityofaustin/atd-geospatial#601]
Using the Mixed Beverage Gross Receipts export from the State of TX, create a Bars and Restaurants point layer for use in Web Maps for SXSW.

**Comment by jaime-mckeown** [comment]
https://comptroller.texas.gov/taxes/mixed-beverage/receipts.php

**Comment by jaime-mckeown** [comment]
- Once you get to the Mixed Beverage Gross Receipts data page, click on View, then Filter the data by Location City of Austin to narrow down the list since it is for all of TX, then download a csv.
- The table will have duplicate records for each location as they record every month. Sort the list by Obligation End Date and remove all records from previous years, up to the current quarter. Most records stopped at 12/31/2019, but some had records for 1/31/2020. Keep only the most recent.
- I then realized that some records had $0 for total receipts. Some of the places had gone out of business, some were remodeling, some were new restaurants that haven't recorded sales yet, and others may not be selling alcohol.
- In order to get a decent geocode, the location address field needs to have the suite removed as it will not match. Create a new field for Suites, and move data over in case it is needed for people in the field.
- Finally geocode the table. I found that addresses on RR, FM, 183, etc. do not geocode as they are entered as the COA Geocoder has locations named differently. Next time I will clean these up in the table before geocoding.
